dashboard/test/models/reference_guide_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class ReferenceGuideTest < ActiveSupport::TestCase
+  test "can create reference guide" do
+    reference_guide = create :reference_guide
+    assert reference_guide.id
+  end
+
+  test "reference guides are unique by slug in course version" do
+    create :reference_guide, key: 'hello/world', course_version_id: 1
+    create :reference_guide, key: 'hello/world', course_version_id: 2
+    assert_raises ActiveRecord::RecordInvalid do
+      create :reference_guide, key: 'hello/world', course_version_id: 1
+    end
+  end
+
+  test "reference guides seed from files correctly" do
+    co = create :course_offering, key: 'course'
+    cv = create :course_version, key: 'version', course_offering: co
+    guide = create :reference_guide, id: '1', display_name: 'World', key: 'hello/world', content: 'hello world', course_version_id: cv.id, position: 2
```

Comment:
This looks like the key for the a reference guide has a slash in it. Is that the plan? That seems like it would be not ideal. @davidsbailey @bethanyaconnor thoughts?

Reply:
oh, oops, that should be an underscore, that is just the test differing

Reply:
wait no, i was thinking of the course_version, key should have a slash, because that is part of the url. will that cause problems somwhere?

Reply:
@tim-dot-org what do `hello` and `world` represent in the key `hello/world`?